### DISCUSSION TITLE
Call to undefined method

### DISCUSSION THREAD

**Discussion by kevinjamau** [noweh/twitter-api-v2-php#27] [General]
My functions were working perfect but today I get error:
Fatal error: Uncaught Error: Call to undefined method Noweh\TwitterApi\Timeline::findRecentMentioningForUserId()

**Comment by noweh** [comment]
hi, a redesign is in progress on the recent tags.

https://github.com/noweh/twitter-api-v2-php/blob/php7.4/src/Timeline.php

This method is now called : getRecentMentions(string $user_id).

**Comment by kevinjamau** [comment]
Thanks for letting me know. Would you please help me to re-write this line
$return = $client->timeline()->findRecentMentioningForUserId('1538300985570885636')->performRequest();
with getRecentMentions?

**Comment by noweh** [comment]
Hello ! a new tag (3.0.0) is available, and the readme.md file has been updated.

With this tag, you have to do : 
    $return = $client->timeline()->getRecentMentions($userId)->performRequest();

Regards
